client/www/pages/_app.tsx

```diff
@@ -140,14 +140,14 @@ function AppHead() {
         content="width=device-width, initial-scale=1, maximum-scale=1"
       />
       <meta name="theme-color" content="#ffffff" />
-      <meta property="og:type" content="website" />
-      <meta property="og:url" content="https://www.instantdb.com" />
-      <meta property="og:title" content="InstantDB: A Modern Firebase" />
+      <meta key="og:type" property="og:type" content="website" />
```

Comment:
I added a `key` to these; this lets child components define a `Head` that overwrites the values when they want too.